src/screens/FinancialSummary/FinancialSummary.tsx

```diff
@@ -82,7 +82,7 @@ export default function FinancialSummary(): JSX.Element {
   }, []);
 
   const getFinancialSummary = async () => {
-    const financialSummary: FinancialSummaryRecord = await getFinancialSummaryById('recMuHpJFsLrNzX01');
+    const financialSummary: FinancialSummaryRecord = await getFinancialSummarieById('recMuHpJFsLrNzX01');
```

Comment:
revert this. The original line here shouldn't compile error if you have the correct `schema.js` and `request.js`